storybook/stories/materials/grid/stories/grid.stories.mdx

```diff
@@ -0,0 +1,222 @@
+import { ExternalLink, TachyonsFile } from "@blocks";
+import { Fragment } from "react";
+import { MaterialsLayout } from "../../materials-layout";
+import { Meta } from "@storybook/addon-docs/blocks";
+import { UtilitiesTable } from "../../shared/utilities-table";
+import { createMaterialSection, paramsBuilder } from "@utils";
+import Installation from "../../installation.mdx";
+
+<Meta
+    title={`${createMaterialSection("Grid")}`}
+    parameters={paramsBuilder()
+        .docsLayout(<MaterialsLayout sourcePath="/packages/tachyons/src" />)
+        .build()}
+/>
+
+# Grid
+
+Grid helps achieve powerful horizontal or vertical layouts without JavaScript. Before using these utilities, you should be familiar with CSS3 Grid specifications. If you are not, check out this [CSS-Tricks guide](https://css-tricks.com/snippets/css/complete-guide-grid/).
+
+A few grid utilities have been added to Tachyons for common use cases that we kept writing specific selectors for.
```

Comment:
A few? In this case I believe we added a new module which is not provided by Tachyons core.